scheduler/src/cook/kubernetes/api.clj

```diff
@@ -890,7 +899,18 @@
     ; init container
     (when use-cook-init?
       (when-let [{:keys [command image]} init-container]
-        (let [container (V1Container.)]
+        (let [container (V1Container.)
+              sidecar-mem (+ computed-mem
```

Comment:
call this total-memory-request and total-memory-limit and add those separately

Reply:
Updated.